content-repo/extra-docs/releases/20.11.0.md

```diff
@@ -0,0 +1,1015 @@
+Maintenance and stability enhancements.
+
+---
+
+### OTRS Service Management XSOAR Pack Pack v1.0.2
+#### Integrations
+##### OTRS  
+- Fixed an issue where the integration created a new session each time a command was executed.
+- Upgraded the Docker image to demisto/pyotrs:1.0.0.12421.
+
+---
+
+### OpenLDAP (Beta) Pack v1.0.2
+#### Integrations
+##### OpenLDAP (Beta)  
+Added the *User Defined Attributes* parameter, which enables you to specify attributes in the user query filter.
+
+---
+
+### OpenPhish Pack v2.0.1
+#### Integrations
+##### New: OpenPhish v2  
+**Breaking Change** url command changed to return multiple entries (entry per indicator) instead of a single entry.
+
+---
+
+### PAN-OS Pack v1.6.4
+#### Integrations
+##### Palo Alto Networks PAN-OS  
+- Documentation and metadata improvements.
+- Fixed an issue where the ***panorama-get-url-category-from-host*** command failed when querying a URL.
+- Updated the Docker image to: *demisto/python3:3.8.6.12176*.
+
+---
+
+### Palo Alto Networks Cortex XDR - Investigation and Response Pack v2.4.8
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response  
+- Maintenance and stability enhancements.
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response rate limit  
+- Improved handling the API rate limit error during the fetch incidents process - The integration will not fail but will create the remaining incidents in the next fetch.
+- Improved handling the API rate limit error in the incoming mirroring process - the sync loop and will be stopped and will resume from the last incident.
+
+---
+
+### Palo Alto Networks PAN-OS EDL Service Pack v1.0.4
+#### Integrations
+##### Palo Alto Networks PAN-OS EDL Service  
+- Fixed an issue where an error was raised if there were no results for the indicators query. 
+- Upgraded the Docker image to demisto/teams:1.0.0.13080.
+
+---
+
+### Phishing Pack v1.10.7
+#### Scripts
+##### PhishingDedupPreprocessingRule  
+- Added arguments that enable setting the threshold to close as duplicate and de-duplication.
+- Default value of *incidentTypes* was changed to 30 days.
+- Added support to recognize emails as duplicate when there is a single word difference.
+
+---
+
+### Prisma Cloud Compute Pack v1.0.4
+#### Integrations
+##### Palo Alto Networks - Prisma Cloud Compute  
+Updated the endpoint that the integration calls.
+
+---
+
+### Qualys Pack v1.0.2
+#### Integrations
+##### Qualys  
+Added the outputs for the ***qualys-vm-scan-fetch*** command.
+
+---
+
+### RSA Archer Pack v1.1.3
+#### Integrations
+##### RSA Archer v2  
+- Improved error handling in login process.
+- Upgraded the Docker image to demisto/python3:3.8.6.12176.
+- You can now pass list fields to the *fieldsToValues* argument without [] in the **archer-create-record** and the **archer-update-record** commands. For example: use "fieldsToValues={"Priority":"High"}" instead of "fieldsToValues={"Priority":"[High]"}".
```

Comment:
```suggestion
- You can now pass list fields to the *fieldsToValues* argument without [] in the **archer-create-record** and the **archer-update-record** commands. For example: use `fieldsToValues={"Priority":"High"}` instead of `fieldsToValues={"Priority":"[High]"}`.
```